formbuilder/components/builder/EditableField.js

```diff
@@ -2,11 +2,18 @@ import React, { Component } from "react";
 import { Draggable, Droppable } from "react-drag-and-drop";
 import Form from "react-jsonschema-form";
 import SchemaField from "react-jsonschema-form/lib/components/fields/SchemaField";
+import { ButtonToolbar, Button} from "react-bootstrap";
+import FieldListDropdown from "./FieldListDropdown"
 
 
 function pickKeys(source, target) {
   const result = {};
   for (let key in source) {
+    if (key =="type") {
+      // type dont have to be update from form data.
+      // it break switchField action.
```

Comment:
Unfortunately I don't understand the comment you entered here. Can you rephrase? Thanks!